configure.ac

```diff
@@ -271,25 +259,92 @@ case $target in
 	  enable_pkey=yes
 	  ;;
      *s390*)
-	  CFLAGS="$CFLAGS -m31 -DNO_PKEY"
+	  CFLAGS="$CFLAGS -m31"
+		AC_DEFINE([NO_PKEY])
```

Comment:
the indention does not seem right for the changed lines. Please align it under the `CFLAGS="$CFLAGS -m31"` line. Same for the other cases.